packages/@adobe/spectrum-css-temp/components/tray/skin.css

```diff
@@ -13,3 +13,10 @@
 .spectrum-Tray {
   background-color: var(--spectrum-tray-background-color);
 }
+
+@media (forced-colors: active) {
+  .spectrum-Tray {
```

Comment:
i don't see what either of these properties are doing, it doesn't appear to change anything?

Reply:
I think I can remove this.